From b3b5fbc67576bf8ecb30bd7ef8771d6c4a76658f Mon Sep 17 00:00:00 2001
Message-ID: <b3b5fbc67576bf8ecb30bd7ef8771d6c4a76658f.1705854493.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705854493.git.sharlatanus@gmail.com>
References: <cover.1705854493.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 15 Jan 2024 20:16:08 +0000
Subject: [PATCH 15/19] gnu: python-skyfield: Update to 1.47.

* gnu/packages/astronomy.scm (python-skyfield): Update to 1.47.
[inputs]: Remove python-certifi, python-jplephem, python-numpy, and python-sgp4.
[propagated-inputs]: Add python-certifi, python-jplephem, python-numpy, and python-sgp4.

Change-Id: If5a9a47077edcebe83449c4c54a49189dea61c5a
---
 gnu/packages/astronomy.scm | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 13046f700a..5adba72439 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4485,20 +4485,23 @@ (define-public python-astroalign
 (define-public python-skyfield
   (package
     (name "python-skyfield")
-    (version "1.46")
+    (version "1.47")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "skyfield" version))
        (sha256
-        (base32 "1r4kpsh1pa4h3diyxy3gyapp0rykfjdqmn5w348a2ck2qkdlx997"))))
+        (base32 "0272pwk88csmrrl44fc4l52agd68pkg4zbmdhs7gliq5q9wf3hrs"))))
     (build-system python-build-system)
     (arguments
-     ;; NOTE: (Sharlatan-20210207T163305+0000): tests depend on custom test
-     ;; framework https://github.com/brandon-rhodes/assay
+     ;; XXX: Tests depend on custom test framework
+     ;; https://github.com/brandon-rhodes/assay
      `(#:tests? #f))
-    (inputs
-     (list python-certifi python-jplephem python-numpy python-sgp4))
+    (propagated-inputs
+     (list python-certifi
+           python-jplephem
+           python-numpy
+           python-sgp4))
     (home-page "https://rhodesmill.org/skyfield/")
     (synopsis "Astronomy for Python")
     (description
-- 
2.41.0

